frontend/src/pages/AccountReceive.vue

```diff
@@ -164,6 +164,7 @@ function useScan() {
 
     // Save off the scanPrivateKeyLocal to memory if it exists, then scan
     if (scanPrivateKeyLocal.value) {
+      if (scanPrivateKeyLocal.value.length === 64) scanPrivateKeyLocal.value = `0x${scanPrivateKeyLocal.value}`;
```

Comment:
This feature (supporting private keys with and without 0x prefix) was originally supported, but we broke it when adding the below `await utils.assertSupportedAddress(computeAddress(scanPrivateKeyLocal.value));` check. This breaks it because `computeAddress` is an ethers.js method that wants the private key to have the 0x prefix.

Previously, we made this 0x change 2 lines below within the `setScanPrivateKey`.

I'd suggest leaving this line as-is, then also:
- Update `setFormStatus` to remove the private key input, since it seems we only call setFormStatus to clear the custom private key anyway
- Update `setScanPrivateKey` to remove the current ``if (key.length === 64) key = `0x${key}` `` check since we now handle that here
- Update `setScanPrivateKey` to validate that the string is either (1) empty, i.e. `''`, or (2) a 66 character hex string

Reply:
> Update setScanPrivateKey to validate that the string is either (1) **empty**

Why do we allow it to be empty?

Reply:
For when you no longer want to use a custom private key!

Reply:
Gotcha, thank you for the thorough explanation!